meshroom/nodes/aliceVision/DepthMap.py

```diff
@@ -63,213 +63,354 @@ class DepthMap(desc.AVCommandLineNode):
+                range=(1, 20, 1),
+                uid=[0],
+            ),
+            desc.IntParam(
+                name='refineWSH',
+                label='WSH',
+                description='Half-size of the patch used to compute the similarity.',
+                value=3,
+                range=(1, 20, 1),
+                uid=[0],
+                advanced=True,
+            ),
+            desc.FloatParam(
+                name='refineSigma',
+                label='Sigma',
+                description='Sigma Threshold.',
+                value=15.0,
+                range=(0.0, 30.0, 0.5),
+                uid=[0],
+                advanced=True,
+            ),
+            desc.FloatParam(
+                name='refineGammaC',
+                label='GammaC',
+                description='GammaC Threshold.',
+                value=15.5,
+                range=(0.0, 30.0, 0.5),
+                uid=[0],
+                advanced=True,
+            ),
+            desc.FloatParam(
+                name='refineGammaP',
+                label='GammaP',
+                description='GammaP threshold.',
+                value=8.0,
+                range=(0.0, 30.0, 0.5),
+                uid=[0],
+                advanced=True,
+            ),
+            desc.BoolParam(
+                name='refineDoRefineFuse',
+                label='Refine and Fuse',
+                description='Perform Refine/Fuse',
+                value=True,
+                uid=[0],
+                advanced=True,
+            ),
+            desc.BoolParam(
+                name='refineDoRefineOptimization',
+                label='Post-Process Optimization',
+                description='Perform Refine post-process optimization',
+                value=True,
+                uid=[0],
+                advanced=True,
+            ),
+        ]),
+        desc.GroupAttribute(
+            name="intermediateResults",
+            label="Intermediate Results",
+            description='Intermediate Results Parameters.',
             advanced=True,
-        ),
-        desc.BoolParam(
-            name='exportIntermediateResults',
-            label='Export Intermediate Results',
-            description='Export intermediate results from the SGM and Refine steps.',
-            value=False,
-            uid=[],
-            advanced=True,
-        ),
+            group=None,
+            groupDesc=[
+            desc.BoolParam(
+                name='exportIntermediateDepthSimMaps',
+                label='Export Depth Maps',
+                description='Export intermediate depth/similarity maps from the SGM and Refine steps.',
+                value=False,
+                uid=[],
+                advanced=True,
+            ),
+            desc.BoolParam(
+                name='exportIntermediateVolumes',
+                label='Export Volumes',
+                description='Export intermediate similarity volumes from the SGM and Refine steps.',
```

Comment:
description='Export intermediate full similarity volumes from the SGM and Refine steps.',